docs/guides/latest/advanced-use-case/pause-backup.md

```diff
@@ -0,0 +1,365 @@
+```
+
+We can see from the above output that the backup session has succeeded. This indicates that the volumes of the Deployment have been backed up in the backend successfully.
+
+## Pause Running Backup
+
+To stop Stash from taking backup volumes of the deployment, you can do the following things:
+
+**Patch BackupConfiguration:**
+
+- Set `spec.paused`: true in `BackupConfiguration` yaml and then apply the update. This happens:
+  - The running backup process will stop immediately and paused `BackupConfiguration` CRD will not be applicable for the targeted workload.
+  - Stash sidecar containers will not be removed from existing workloads but the sidecar will stop taking backup.
+
+You can do the things by using the following command,
+
+```console
+$ kubectl patch backupconfiguration -n demo stash-demo --type="merge" --patch='{"spec": {"paused": true}}'
+backupconfiguration.stash.appscode.com/pause-backup patched
+```
+
+Now, Describe the `BackupConfiguration` crd by using the following command,
+
+```yaml
+$ kubectl describe backupconfiguration -n demo pause-backup
+Name:         pause-backup
+Namespace:    demo
+Labels:       <none>
+Annotations:  kubectl.kubernetes.io/last-applied-configuration:
+                {"apiVersion":"stash.appscode.com/v1beta1","kind":"BackupConfiguration","metadata":{"annotations":{},"name":"pause-backup","namespace":"de...
+API Version:  stash.appscode.com/v1beta1
+Kind:         BackupConfiguration
+Metadata:
+  Creation Timestamp:  2019-08-01T11:42:22Z
+  Finalizers:
+    stash.appscode.com
+  Generation:        2
+  Resource Version:  14465
+  Self Link:         /apis/stash.appscode.com/v1beta1/namespaces/demo/backupconfigurations/pause-backup
+  UID:               5e1abe4b-c3d2-438a-ab8d-67ed7e663546
+Spec:
+  Paused:  true
+  Repository:
+    Name:  gcs-repo
+  Retention Policy:
+    Keep Last:  5
+    Name:       keep-last-5
+    Prune:      true
+  Schedule:     */1 * * * *
+  Target:
+    Directories:
+      /source/data
+    Ref:
+      API Version:  apps/v1
+      Kind:         Deployment
+      Name:         stash-demo
+    Volume Mounts:
+      Mount Path:  /source/data
+      Name:        source-data
+Events:
+  Type    Reason          Age   From                       Message
+  ----    ------          ----  ----                       -------
+  Normal  Backup Skipped  25s   Backup Triggering CronJob  Skipping creating BackupSession. Reason: Backup Configuration demo/pause-backup is paused.
+```
+
+Look at the `Events` of the `BackupConfiguration` crd. The event shows that the creation of `BackupSession` has paused. This means the hole backup process has stopped.
+
+**Skipped Instant Backup:**
+
+If we try to create a `BackupSession` for instant backup  the `BackupSession` will be `Skipped`. Now, we are going to create a `BackupSession` crd for instant backup. Below is the YAML of the `BackupSession` that we are going to create,
```

Comment:
If we try to trigger an instant backup by creating a `BackupSession`, it will get skipped. The sidecar container will write an event to the BackupSession describing why it has skipped taking the backup.

Below is the YAML of the `BackupSession` that we are going to create,